src/iris/api.py

```diff
@@ -684,7 +685,16 @@ def debug_auth(self, req, resp, resource, params):
 
         # Proceed with authenticating this route as a third party application
         try:
-            app, client_digest = req.get_header('AUTHORIZATION', '')[5:].split(':', 1)
+            # Ignore HMAC requirements for custom webhooks
+            if len(req.env['PATH_INFO'].split("/")) > 2 and req.env['PATH_INFO'].split("/")[2] == 'webhooks':
```

Comment:
Can we do `if req.env['PATH_INFO'].startswith('/v0/webhooks/')` instead? Should be more efficient than splitting twice and it's less code.